include/zephyr/bluetooth/audio/codec.h

```diff
@@ -0,0 +1,1186 @@
+ *
+ *  @param _loc             Audio channel location bitfield (@ref bt_audio_location)
+ *  @param _stream_context  Stream context (BT_AUDIO_CONTEXT_*)
+ */
+#define BT_AUDIO_LC3_BROADCAST_PRESET_48_1_2(_loc, _stream_context) \
+	BT_AUDIO_LC3_PRESET( \
+		BT_CODEC_LC3_CONFIG_48_1(_loc, _stream_context), \
+		BT_CODEC_LC3_QOS_7_5_UNFRAMED(75u, 4u, 50u, 40000u) \
+	)
+
+/**
+ *  @brief Helper to declare LC3 Broadcast 48_2_2 codec configuration
+ *
+ *  @param _loc             Audio channel location bitfield (@ref bt_audio_location)
+ *  @param _stream_context  Stream context (BT_AUDIO_CONTEXT_*)
+ */
+#define BT_AUDIO_LC3_BROADCAST_PRESET_48_2_2(_loc, _stream_context) \
+	BT_AUDIO_LC3_PRESET( \
+		BT_CODEC_LC3_CONFIG_48_2(_loc, _stream_context), \
+		BT_CODEC_LC3_QOS_10_UNFRAMED(100u, 4u, 65u, 40000u) \
+	)
+
+/**
+ *  @brief Helper to declare LC3 Broadcast 48_3_2 codec configuration
+ *
+ *  @param _loc             Audio channel location bitfield (@ref bt_audio_location)
+ *  @param _stream_context  Stream context (BT_AUDIO_CONTEXT_*)
+ */
+#define BT_AUDIO_LC3_BROADCAST_PRESET_48_3_2(_loc, _stream_context) \
+	BT_AUDIO_LC3_PRESET( \
+		BT_CODEC_LC3_CONFIG_48_3(_loc, _stream_context), \
+		BT_CODEC_LC3_QOS_7_5_UNFRAMED(90u, 4u, 50u, 40000u) \
+	)
+
+/**
+ *  @brief Helper to declare LC3 Broadcast 48_4_2 codec configuration
+ *
+ *  @param _loc             Audio channel location bitfield (@ref bt_audio_location)
+ *  @param _stream_context  Stream context (BT_AUDIO_CONTEXT_*)
+ */
+#define BT_AUDIO_LC3_BROADCAST_PRESET_48_4_2(_loc, _stream_context) \
+	BT_AUDIO_LC3_PRESET( \
+		BT_CODEC_LC3_CONFIG_48_4(_loc, _stream_context), \
+		BT_CODEC_LC3_QOS_10_UNFRAMED(120u, 4u, 65u, 40000u) \
+	)
+
+/**
+ *  @brief Helper to declare LC3 Broadcast 48_5_2 codec configuration
+ *
+ *  @param _loc             Audio channel location bitfield (@ref bt_audio_location)
+ *  @param _stream_context  Stream context (BT_AUDIO_CONTEXT_*)
+ */
+#define BT_AUDIO_LC3_BROADCAST_PRESET_48_5_2(_loc, _stream_context) \
+	BT_AUDIO_LC3_PRESET( \
+		BT_CODEC_LC3_CONFIG_48_5(_loc, _stream_context), \
+		BT_CODEC_LC3_QOS_7_5_UNFRAMED(117u, 4u, 50u, 40000u) \
+	)
+
+/**
+ *  @brief Helper to declare LC3 Broadcast 48_6_2 codec configuration
+ *
+ *  @param _loc             Audio channel location bitfield (@ref bt_audio_location)
+ *  @param _stream_context  Stream context (BT_AUDIO_CONTEXT_*)
+ */
+#define BT_AUDIO_LC3_BROADCAST_PRESET_48_6_2(_loc, _stream_context) \
+	BT_AUDIO_LC3_PRESET( \
+		BT_CODEC_LC3_CONFIG_48_6(_loc, _stream_context), \
+		BT_CODEC_LC3_QOS_10_UNFRAMED(155u, 4u, 65u, 40000u) \
+	)
+
+/**
+ * @brief Audio codec Config APIs
+ * @defgroup bt_audio_codec_cfg Codec config parsing APIs
+ *
+ * Functions to parse codec config data when formatted as LTV wrapped into @ref bt_codec.
+ *
+ * @{
+ */
+
+/**
+ * @brief Codec parser error codes for @ref bt_audio_codec_cfg.
+ */
+enum bt_audio_codec_parse_err {
```

Comment:
You have right, I saw that, but I did not want to make any further changes (changing the naming) since I did not know this change will be accepted. If we agree the code can be moved to this file, I'll add another patch on top to refactor the naming.